### PR TITLE
RGO2018

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
I approve this PR.

File: .gitignore [hunk 1]
```diff
@@ -63,3 +63,4 @@ target/
 
 # PyCharm
 .idea
+deps/
```
Comment (Rayman) [inline .gitignore]: Deps is the folder with submodules so it should not be ignored

Comment (MatthijsBurgh) [reply .gitignore]: But these should never be pushed to git right?

Comment (Rayman) [reply .gitignore]: If git says there are modifications, then you DONT have the version which this package specifies. Don't hide that information, it's important.

Comment (MatthijsBurgh) [reply .gitignore]: I remembered that it did show already after a clean install. Therefore I added it to the gitignore. But now I did a clean recursive clone, I doesn't show. So I will revert.

**Pull request merged**